### ISSUE TITLE
The layour of Markdown with images does not work well

### ISSUE THREAD

**Issue by MarcSkovMadsen** [bokeh/bokeh#9538]
FIRST POSTED TO THE PANEL ISSUE LIST HERE. 

@philippjfr SUGGESTED POSITING IT HERE.

#### System Info

- Panel: 0.7.0
- Bokeh: 1.4.0
- Python: 3.7.4
- OS: Windows 8.1
- Browser: Chrome

#### Description of expected behavior and the observed behavior

If I do a simple layout using columns and markdown with an image I would expect it to display nicely. 

It does not. Instead it displays on top of each other.

![image](https://user-images.githubusercontent.com/42288570/70017590-d106bc00-1583-11ea-9dfb-8d5b5f3f4399.png)

#### Complete, minimal, self-contained example code that reproduces the issue

```python
import panel as pn

ABOUT = """\
# About

[Panel](https://panel.pyviz.org/) is a powerfull framework for building awesome-analytics apps in [Python](https://www.python.org/).

The purpose of this app is to test that a **multi-page Dashboard Layout** similar to the [bootstrap dashboard template](https://getbootstrap.com/docs/4.3/examples/dashboard/) from [getboostrap.com](https://getbootstrap.com/) can be implemented in [Panel](https://panel.pyviz.org/).

[![GetBootstrap Dashboard Template](https://getbootstrap.com/docs/4.4/assets/img/examples/dashboard.png)](https://getbootstrap.com/docs/4.3/examples/dashboard/)
"""

INFO = """\
Navigate to the **Dashboard Page** via the **Sidebar** to see the result.
Or Navigate to the **Limitations Page** to learn of some of the limitations of Panel that
I've experienced."""

about = pn.pane.Markdown(ABOUT)
info = pn.pane.Markdown(INFO, background="#d1ecf1")
app = pn.Column(about, info)
app.servable()
```

### Additional Context

If I then resize or refresh the window. Then it displays correctly.

![issue_markdown_w_image_markdown](https://user-images.githubusercontent.com/42288570/70017452-6ce3f800-1583-11ea-98e7-f47204b77d33.gif)

I have a feeling the Bokeh javascript some how lays out the page before actual knowing that there is a image taking up some space. But that's the feeling.

**Comment by philippjfr** [comment]
>I have a feeling the Bokeh javascript some how lays out the page before actual knowing that there is a image taking up some space

This is correct, it doesn't know when the DOM content is finished loading. We have discussed some solutions for this but @mattpap will be able to describe them better.

Note that bokeh issues should use bokeh code to demonstrate the issue, e.g. bokeh does not know anything about markdown so the real issue is simply HTML images. At some point I actually had a basic Panel object -> Bokeh code converter which would make this easier. So here's a simple example to demonstrate the issue:

```python
from bokeh.models import Div, Column
from bokeh.plotting import show

ABOUT = """<h1>About</h1>\n<p><a href="https://panel.pyviz.org/">Panel</a> is a powerfull framework for building <strong>awesome analytics apps</strong>
in <a href="https://www.python.org/">Python</a>.</p>\n<p>The purpose of this app is to test that a <strong>multi-page Dashboard Layout</strong>
similar to the <a href="https://getbootstrap.com/docs/4.3/examples/dashboard/">bootstrap dashboard template</a> from
<a href="https://getbootstrap.com/">getboostrap.com</a> can be implemented in <a href="https://panel.pyviz.org/">Panel</a>.</p>
"""

IMG = '<img src="https://cdn.shortpixel.ai/spai/w_300+q_lossy+ret_img+to_webp/https://www.numfocus.org/wp-content/uploads/2017/11/bokeh-logo-300.png"></img>'

about = Div(text=ABOUT)
img = Div(text=IMG)

col = Column(children=[img, about])

show(col)
```

![Screen Shot 2019-12-23 at 2 48 29 PM](https://user-images.githubusercontent.com/1550771/71361489-639ee780-2593-11ea-86ea-f61ead5b3bc4.png)

**Comment by bryevdv** [comment]
@philippjfr @mattpap is this also something  covered by porting the HTML component from panel? can someone poitnt me at that work/PR?

**Comment by philippjfr** [comment]
No, this isn't covered by that. In fact if this were implemented the trick used by the HTML model would likely not work. Here's the code that implements the caching:

https://github.com/holoviz/panel/blob/master/panel/models/html.ts#L14

**Comment by MarcSkovMadsen** [comment]
Any updates on this as this is really a bummer?
